src/list-sorting/index.js

```diff
@@ -1,3 +1,21 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+    if (typeof haystack[0] != "object"){
+        return haystack.lastIndexOf(needle)
+    }
+
+    let row = 0
+    let column = 0
+    let match = false
+
+    haystack.forEach((element, index, array) => {
+      if (element.includes(needle)) {
+        row = index
+        column = element.lastIndexOf(needle)
+        match = true
+      }
+    })
+
+    return !match ? -1 : [row, column]
```

Comment:
A reversed condition would be easier to read. Easy to skip over the `!` sometimes.

`match ? [row, column] : -1`

Reply:
Noted